app/views/content/subjects/maths/_article.html.erb

```diff
@@ -36,7 +36,7 @@
     </p>
   </section>
 
-  <%= render SubjectSpecific::TextBlockWithTwoImagesComponent.new(
+  <%= render Campaign::TextBlockWithTwoImagesComponent.new(
```

Comment:
These images look much better on desktop now - still a little small on mobile compared with prod, so they lose a bit of impact I think?

Reply:
I'll make these a bit bigger but IMO they're too big in prod and almost hit the edge of the page for mobile; they should have our standard margin I think to be inline with the text below (although they'll be slightly off that due to the rotation!)